MonoGame.Framework/Platform/Utilities/FuncLoader.Desktop.cs

```diff
@@ -65,6 +65,17 @@ public static IntPtr LoadLibraryExt(string libname)
             if (ret == IntPtr.Zero)
                 ret = LoadLibrary(Path.Combine(assemblyLocation, libname));
 
+            // Try MacOS folder for macOS .app bundles
+            if (ret == IntPtr.Zero)
+            {
+                // Correct the working directory location for macOS .app bundles
+                // App's current working directory is actually `/`, so "./" won't work.
+                var currentDirectory = System.AppDomain.CurrentDomain.BaseDirectory;
+
+                // Look in MacOS for .app bundles
+                ret = LoadLibrary(Path.Combine(currentDirectory, libname));
```

Comment:
You are missing `if (ret == IntPtr.Zero)` check.

Reply:
I'm sure I have the check.

Reply:
Oh yea, you didn't check for macos with the line above, this is fine.